esp/xslt/esdl2monitor.xslt

```diff
@@ -21,6 +21,7 @@
     <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
     <xsl:param name="responseType" select="''"/>
     <xsl:param name="requestType" select="''"/>
+    <xsl:param name="platform" select="'roxie'"/>
```

Comment:
Minor, but it looks like the other platform type used is "ESP" all upper, we should be consistent "ROXIE" or "Roxie"